src/pages/dashboard/unstake-banner/unstake-banner.module.scss

```
@@ -1,44 +1,52 @@
 @import '../../../styles/variables.module.scss';
+@import '../../../styles/fonts.module.scss';
 
 .unstakeBanner {
   width: 100%;
   display: flex;
+  flex-direction: column;
+  gap: 16px;
   align-items: center;
   justify-content: space-between;
-  padding: $space-lg 0;
-  border-top: 1px solid rgba(124, 227, 203, 0.4);
-  border-bottom: 1px solid rgba(124, 227, 203, 0.4);
-  margin-bottom: 2 * $space-xl;
-
-  @media (max-width: $max-md) {
-    flex-direction: column;
-
-    & > *:not(:last-child) {
-      margin-bottom: $space-lg;
-    }
+  background: linear-gradient(180deg, #fafafa 0%, #f0f4ff 100%);
+  border-top: 1px solid $color-dark-blue-10;
+  border-bottom: 1px solid $color-dark-blue-10;
+  margin: 32px 0;
+  padding: 24px 16px;
+
+  @media (min-width: $md) {
```

Actually I had a comment, not sure why it wasn't recorded when I approved the PR.

I was suggesting this change, which looks better imo

```suggestion
  @media (min-width: $sm) {
```

---

it's better indeed